src/app.js

```diff
@@ -103,7 +103,8 @@ document.addEventListener('DOMContentLoaded', function () {
 });
 
 document.addEventListener('copy', copySelectedToClipboard, true);
-document.addEventListener('deselectAll', function() { global.fileList.Select.deselectAll(); }, true);
+document.addEventListener('removeSelected', global.fileList.Select.removeSelected(), true);
```

Comment:
This has to be `document.addEventListener('removeSelected', global.fileList.Select.removeSelected, true);`